File: Fundamentos da Programação Orientada a Objetos/HashSet/ex01/principal/Principal.java
package principal;

import entidades.Estoque;
import entidades.Produto;


public class Principal {
    public static void main(String[] args) {
     
        Produto produto_1 = new Produto(1, "Camisa", 15);
        Produto produto_2 = new Produto(2, "Bone", 10);
        Produto produto_3 = new Produto(3, "Meias", 5);
        
        Estoque estoque = new Estoque();
        estoque.adicionarProduto(produto_1);
        estoque.adicionarProduto(produto_2);
        estoque.adicionarProduto(produto_3);

        estoque.listarProdutos();
        estoque.removerProduto(2);
        estoque.listarProdutos();

        Produto produto_4 = estoque.buscarProduto(1);
        if (produto_4 != null) {
        	produto_4.setQuantidade(15);
        }

        estoque.listarProdutos();
    }
}

File: Programação Web Front-End/Lista_Array_POO/exerc_01/js/Principal.js
import Produto from './Produto.js';
import Produtos from './Produtos.js';

let produtos = new Produtos();

// Adicionando produtos
let produto1 = new Produto("Arroz", "123", "01/08/2023");
produtos.adicionarProduto(produto1);

let produto2 = new Produto("Leite", "234", "22/05/2024");
produtos.adicionarProduto(produto2);

let produto3 = new Produto("Leite", "345", "22/05/2024");
produtos.adicionarProduto(produto3);

let produto4 = new Produto("Feijão", "234", "22/05/2024");
produtos.adicionarProduto(produto4);



// Removendo produtos por índice
produtos.removerProdutoPorIndice(0);

// Removendo produtos por nome
produtos.removerProdutoPorNome("Arroz");

// Consultando quantidade por nome
produtos.consultarQuantidadePorNome("Leite");

// listar todos os produtos existentes
produtos.listarProdutos();